eggplant/membership/tests.py

```diff
@@ -162,7 +158,7 @@ def test_send_invitation(self):
         expected = 'Invitation has been send to {}'.format(invited_email)
         self.assertContains(response, expected, 1, 200)
 
-        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(len(mail.outbox), 1)  # @UndefinedVariable
```

Comment:
They're really really helpful because I usually spot many errors by having a static check enforced through-out the codebase (especially from people who don't use a static check). It's because mail.outbox doesn't resolve, but I think it's like the only place in the whole codebase there's something like that :)